static/app/bootstrap/initializeSdk.tsx

```diff
@@ -58,6 +59,18 @@ function getSentryIntegrations(sentryConfig: Config['sentryConfig'], routes?: Fu
     }),
   ];
 
+  if (sentryConfig.replaysSessionSampleRate || sentryConfig.replaysOnErrorSampleRate) {
+    integrations.push(
+      new Sentry.Replay({
+        maskAllText: true,
+        _experiments: {
+          captureExceptions: true,
+          traceInternals: true,
```

Comment:
What are these options controlling?

Reply:
They ensure we capture exceptions & internals from replay to sentry itself. Otherwise, by default we ignore any errors that happen inside of replay from a users perspective.